<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>430</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sistema de Controle de Estoque</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::EditSelectAll"/>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>361</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="botaoImportar">
       <property name="text">
        <string>Importar arquivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoAdicionar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Adicionar item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoRemover">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remover item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoGerarRelatorio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Gerar relat√≥rio (.txt)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
